web/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -102,6 +105,9 @@ class Panel extends Component<PanelProps, PanelState> {
     this.executeQuery();
   }
 
+  // eslint-disable-next-line
+  debounceExecuteQuery() {}
```

Comment:
Could we declare this more like we do for `abortInFlightFetch` above?

Reply:
Definitely. It should be like that.